docs/release/releases/release-1.9.md

```diff
@@ -0,0 +1,36 @@
+# Cluster API v1.9
+
+## Timeline
+
+The following table shows the preliminary dates for the `v1.9` release cycle.
+
+| **What**                                             | **Who**      | **When**                    | **Week** |
+|------------------------------------------------------|--------------|-----------------------------|----------|
+| Start of Release Cycle                               | Release Lead | Monday 19th August 2024     | week 1   |
+| Schedule finalized                                   | Release Lead | Friday 23rd August 2024     | week 1   |
+| Team finalized                                       | Release Lead | Friday 23rd August 2024     | week 1   |
+| *v1.7.x & v1.8.x released*                           | Release Lead | Tuesday 10th September 2024 | week 4   |
+| *v1.7.x & v1.8.x released*                           | Release Lead | Tuesday 8th October 2024    | week 8   |
+| *v1.7.x & v1.8.x released*                           | Release Lead | Tuesday 5th November 2024   | week 12  |
+| v1.9.0-beta.0 released                               | Release Lead | Tuesday 5th November 2024   | week 12  |
+| Communicate beta to providers                        | Release Lead | Tuesday 5th November 2024   | week 12  |
+| KubeCon idle week                                    | Release Lead | Tuesday 12th November 2024  | week 13  |
+| v1.9.0-beta.x released                               | Release Lead | Tuesday 19th November 2024  | week 14  |
+| release-1.8 branch created (**Begin [Code Freeze]**) | Release Lead | Tuesday 26th November 2024  | week 15  |
+| v1.9.0-rc.0 released                                 | Release Lead | Tuesday 26th November 2024  | week 15  |
+| release-1.8 jobs created                             | Release Lead | Tuesday 26th November 2024  | week 15  |
+| v1.9.0-rc.x released                                 | Release Lead | Tuesday 3rd December 2024   | week 16  |
+| **v1.9.0 released**                                  | Release Lead | Tuesday 10th Decemeber 2024 | week 17  |
+| *v1.7.x & v1.8.x released*                           | Release Lead | Tuesday 10th Decemeber 2024 | week 17  |
```

Comment:
```suggestion
| **v1.9.0 released**                                  | Release Lead | Tuesday 10th December 2024 | week 17  |
| *v1.7.x & v1.8.x released*                           | Release Lead | Tuesday 10th December 2024 | week 17  |
```

Let's also add a tentative 1.9.1 release one week later (like in the release-1.8.md